server/jetty-app/README.md

```diff
@@ -1,90 +1,82 @@
-    ```shell
-    $ mkdir dh-py && cd dh-py
-    $ python -m venv local-jetty-build
-    $ source local-jetty-build/bin/activate # this must be re-run for each new shell
-    $ cd -
-   ```
-1. Build and install wheels for deephaven-jpy and deephaven:
-    ```shell
-    $ python -m pip install --upgrade pip # First upgrade pip
-    $ pip install wheel
-    $ export DEEPHAVEN_VERSION=0.21.0 # this should match the current version of your git repo
-
-    $ cd py/jpy
-    $ export JAVA_HOME=/path/to/your/java/home # Customize this to fit your computer
-    $ python setup.py bdist_wheel
-    $ pip install dist/deephaven_jpy-0.21.0-cp39-cp39-linux_x86_64.whl # This will vary by version/platform
-    $ cd -
+# server-jetty-app
 
-    $ cd Integrations/python
-    $ python setup.py bdist_wheel
-    $ pip install dist/deephaven-0.21.0-py2.py3-none-any.whl
-    $ cd -
-    ```
+## Native development
 
+This document is oriented towards getting a server up for local development.
+If you are interested in running a native production release, please see
+[https://deephaven.io/core/docs/how-to-guides/configuration/native-application/](https://deephaven.io/core/docs/how-to-guides/configuration/native-application/).
 
-### Build
+### TLDR Groovy
 
 ```shell
-./gradlew server-jetty-app:build
+./gradlew server-jetty-app:run -Pgroovy 
 ```
 
-produces
+### TLDR Python
 
-* `server/jetty-app/build/distributions/server-jetty-<version>.tar`
-* `server/jetty-app/build/distributions/server-jetty-<version>.zip`
+1. Setup virtual environment:
 
-### Run
+   ```shell
+   # Note: 
+   python -m venv /tmp/my-dh-venv
+   source /tmp/my-dh-venv/bin/activate
+   ```
 
-The above artifacts can be uncompressed and their `bin/start` script can be executed:
+1. Build wheel
 
-```shell
-START_OPTS="-Ddeephaven.console.type=groovy" bin/start
-```
+   ```shell
+   ./gradlew py-server:assemble
+   # replace with the appropriate <version>
+   pip install "py/server/build/wheel/deephaven_core-<version>-py3-none-any.whl[autocomplete]"
+   ```
+
+1. Run
 
-Alternatively, the uncompressed installation can be built directly by gradle:
+   ```shell
+   ./gradlew server-jetty-app:run
+   ```
+
+A couple notes:
+
+* Depending on your OS and how your PATH is setup, you may need to use `python3`, or a path to the explicit python version you want to use
+* You may choose to setup a "permanent" virtual environment location
+* You don't need to install the `[autocomplete]` feature if you don't want to
+* You'll need to re-install the wheel anytime you are making python code changes that effect the wheel
+* `pip` can be a pain if you are trying to (re-)install a wheel with the same version number as before
+  * A `pip install --force-reinstall --no-deps "py/server/build/wheel/deephaven_core-<version>-py3-none-any.whl[autocomplete]"` may do the trick
+* You can install other python packages
+
+### Start script
+
+You can create and invoke the start script instead of using a gradle exec task:
```

Comment:
Why would I want to do this? Also slightly confusing because you say `instead of using a gradle exec task` and then the first line is a gradle command (though I guess it's not an exec task?)